### PR TITLE
Create chat group

### PR DESCRIPTION
**Pull request by jephillips**
Added a dialog that will create a new chat group and add it to the Group chat view
You currently must use the Quickblox user ids seperated by commas (i.e. (80001,90001,10001) to add users until we implement the contacts sheet selection

### CODE REVIEW

File: brewchat/src/main/java/com/example/brewchat/ChatService.java [hunk 1]
```diff
@@ -58,7 +62,7 @@ public ChatService(Context context, String appId, String authKey, String authSec
 
         QBSettings.getInstance().fastConfigInit(appId, authKey, authSecret);
 
-//        QBChatService.setDebugEnabled(true);
+        QBChatService.setDebugEnabled(true);
```
Comment (arreche) [inline brewchat/src/main/java/com/example/brewchat/ChatService.java]: Enable this on debug

``` java
if (BuildConfig.DEBUG) {
    QBChatService.setDebugEnabled(true);
}
```

OR

``` java
QBChatService.setDebugEnabled(BuildConfig.DEBUG);
```

File: brewchat/src/main/java/com/example/brewchat/activities/ChatManagerActivity.java [hunk 2]
```diff
@@ -48,11 +78,20 @@ public boolean onOptionsItemSelected(MenuItem item) {
         //noinspection SimplifiableIfStatement
         if (id == R.id.action_settings) {
             return true;
+        }else if(id == R.id.action_add_group_chat){
```
Comment (arreche) [inline brewchat/src/main/java/com/example/brewchat/activities/ChatManagerActivity.java:87]: Missing space between `}` and `else`

File: brewchat/src/main/java/com/example/brewchat/ChatService.java (hunk 1 above)
Comment (JaySeeJC) [inline brewchat/src/main/java/com/example/brewchat/ChatService.java]: +1

File: brewchat/src/main/java/com/example/brewchat/ChatService.java [hunk 3]
```diff
@@ -115,6 +122,29 @@ public void logout() {
...
+            @Override
+            public void onSuccess(QBDialog dialog, Bundle args) {
+                GroupChatCreatedEvent groupChatCreatedEvent = new GroupChatCreatedEvent(dialog.getName(), dialog.getOccupants());
+                EventBus.getDefault().post(groupChatCreatedEvent);
+                Log.d(TAG, "Created new Group Chat");
+            }
+
+            @Override
+            public void onError(List<String> errors) {
+                //Toast saying error in creating group chat
```
Comment (arreche) [inline brewchat/src/main/java/com/example/brewchat/ChatService.java]: Mark this comments with `TODO` or `FIXME` so we can search for them latter.

Please do not use a toast in this class. :)

**Comment by cjkennedyo** [comment]
+1

**Pull request merged**